library/ssl_tls.c

```diff
@@ -9972,6 +9977,64 @@ int mbedtls_ssl_get_key_exchange_md_ssl_tls( mbedtls_ssl_context *ssl,
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1) || defined(MBEDTLS_SSL_PROTO_TLS1_1) || \
     defined(MBEDTLS_SSL_PROTO_TLS1_2)
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+int mbedtls_ssl_get_key_exchange_md_tls1_2( mbedtls_ssl_context *ssl,
+                                            unsigned char *hash, size_t *hashlen,
+                                            unsigned char *data, size_t data_len,
+                                            mbedtls_md_type_t md_alg )
+{
+    int ret = 0;
+    psa_hash_operation_t hash_operation;
```

Comment:
This must be initialized; see https://github.com/ARMmbed/mbedtls-psa/blob/feature-psa/include/psa/crypto.h#L707.

Reply:
Unfortunately hash initialization is introduced in a later version of crypto.h than used in this PR. Would you like me to change that, or rather create a bigger PR later, updating other library uses as well?

Reply:
No, we should only update the version of PSA Crypto in dedicated PRs. If hash initialization wasn't yet introduced in the version we're using here, your code is fine.